content/css/concepts/flexbox/terms/flex-basis/flex-basis.md

```diff
@@ -45,3 +45,6 @@ An image that has a starting width of 200 pixels:
   flex-basis: 200px;
 }
 ```
+The output of the above codeblock is shown below:
+
+![Image demonstrates the use of 200px `flex-basis` on `#img_one` alongside other images in the flex with default properties.](../../../../../../media/css-flex-basis-example-output.png)
```

Comment:
Also, change the image link to the following format:
"https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.